config/sidekiq_cron_schedule.yml

```diff
@@ -0,0 +1,24 @@
+session_trim:
+  cron: "0 1 * * *"
+  class: "SessionTrimJob"
+  queue: default
+create_new_fake_sandbox_data:
+  cron: "0 2 * * *"
+  class: "CreateNewFakeSandboxDataJob"
+  queue: default
```

Comment:
I know this early returns on prod, but this still looks a little scary. Would it be worth having one schedule per environment do you think?